packages/relay/src/lib/eth.ts

```diff
@@ -1566,8 +1565,12 @@ export class EthImpl implements Eth {
    * @param gas
    * @param requestIdPrefix
    */
-  async callConsensusNode(call: any, gas: number, requestIdPrefix?: string): Promise<string | JsonRpcError> {
+  async callConsensusNode(call: any, gas: number | null, requestIdPrefix?: string): Promise<string | JsonRpcError> {
     // Execute the call and get the response
+    if (!gas) {
+      gas = Number.parseInt(this.defaultGas);
```

Comment:
So we do need to set gas for calls to consensus node, but not mirror node?

Reply:
Not sure if it's needed. But we ultimatly have to remove the fallback to consensus at one point, so it doesn't really matter.

Reply:
agreed 👍